Content.Client/Consent/UI/Windows/ConsentWindow.xaml.cs

```diff
@@ -157,7 +157,10 @@ public void UpdateUi()
 
         _entries.Clear();
 
-        var consentprototypelist = _protoManager.EnumeratePrototypes<ConsentTogglePrototype>();
+        var consentprototypelist = new List<ConsentTogglePrototype>(_protoManager.EnumeratePrototypes<ConsentTogglePrototype>());
+
+        consentprototypelist.Sort();
+
```

Comment:
Code style: use the .ToList extension method:
```suggestion
        var consentprototypelist = _protoManager.EnumeratePrototypes<ConsentTogglePrototype>().ToList();
        consentprototypelist.Sort();

```

Reply:
This data type doesn't have a ToList function, but System.Linq.Enumerable has a ToList<T>(IEnumerable<T> source) function if that's acceptable

Reply:
yeah, your IDE is supposed to import that method automatically (if you are using an IDE). We don't typically type out imports ourselves.

Reply:
Yeah, since most of the changes I've made so far were just yml or minor code changes I haven't bothered setting up an IDE, just using Sublime with syntax highlighting, but I am starting to poke around more with the code so that'll probably be worth the effort to set up soon.